thrift/src/main/thrift/atoms/sidenote.thrift

```diff
@@ -0,0 +1,58 @@
+namespace * contentatom.sidenote
+namespace java com.gu.contentatom.thrift.atom.sidenote
+#@namespace scala com.gu.contentatom.thrift.atom.sidenote
+
+include "entity.thrift"
+include "../shared.thrift"
+
+enum SideNoteType {
+  QA = 0,
+  GUIDE = 1,
+  TIMELINE = 2,
+  PROFILE = 3
+}
+
+struct SideNoteAtom {
+  1: optional string typeLabel
+  2: required string title
+  3: optional list<string> tags
+
+  4: optional SideNoteType SideNoteType
```

Comment:
This should be required - to create a SideNote it's imperative you know what type it is.